lib/msgPackPeer.ts

```diff
@@ -1,9 +1,10 @@
-import { Peer, type SerializerMapping } from "./peer";
-import { MsgPack } from "./exports";
+import { Peer, } from "./peer";
+import { MsgPack} from "./exports";
+import { SerializerMapping, defaultSerializers } from "./optionInterfaces";
 
 export class MsgPackPeer extends Peer {
 	override _serializers: SerializerMapping = {
-		MsgPack,
+		...defaultSerializers,
```

Comment:
Ah I see, thats very fair, in that case, I think it would be more appropriate just to force the user to dependency inject the serialization mapping than to make permuations of `Peer` that contain the serializers.

This would mean a 'blank' Peer that contains no serializers to begin with, but im not sure how the tree shaking would work on that.